keyboards/neopad/rev1/keymaps/default/keymap.c

```diff
@@ -16,79 +16,76 @@
+     * |  TO(1)  |   up    |   mute  |	L ENC: brightness up/down
+     * |---------+---------+---------|
+     * |   left  |  down   |  right  |	R ENC: volume up/down
+     * `-----------------------------'
+     */
     [0] = LAYOUT(
-                TO(1),     KC_MSTP,  KC_MUTE,
-    		    KC_MPRV,   KC_MPLY,  KC_MNXT
+                TO(1),     KC_UP,    KC_MUTE,
+    		    KC_LEFT,   KC_DOWN,  KC_RIGHT
     ),
 
-    /*
-            LAYER 1 - MOVEMENT IN WINDOWS
-    /---------------------------------------`
-    |    TO(2)    | maximize | show desktop | L ENC: change desktop
-    |-------------|----------|--------------|
-    | move window | minimize | move window  | R ENC: change browser tab + change explorer window
-    \---------------------------------------'
-    */
+    /* LAYER 1
+     * ,---------------------------------------.
+     * |    TO(2)    | maximize | show desktop | L ENC: page up/down
+     * |-------------+----------+--------------|
+     * | move window | minimize | move window  | R ENC: mouse wheel up/down
+     * `---------------------------------------'
+     */
     [1] = LAYOUT(
 				TO(2),     		LGUI(KC_UP),    LGUI(KC_D),
 				LGUI(KC_LEFT),	LGUI(KC_DOWN),  LGUI(KC_RIGHT)
     ),
 
-    /*
-            LAYER 2 - SHORTCUT
-    /------------------------------`
-    |  TO(3)  |   esc   | task man | L ENC: redo/undo
-    |---------|---------|----------|
-    |   cut   |   copy  |   paste  | R ENC: mouse wheel up/down
-    \------------------------------'
-    */
+    /* LAYER 2
+     * ,------------------------------.
+     * |  TO(3)  |   esc   | sel all  | L ENC: redo/undo
+     * |---------+---------+----------|
+     * |   cut   |   copy  |   paste  | R ENC: forward/backward one word
+     * `------------------------------'
+     */
     [2] = LAYOUT(
-                TO(3),     		KC_ESC,    		LCTL(LSFT(KC_ESC)),
+                TO(3),     		KC_ESC,    		LCTL(KC_A),
     		    LCTL(KC_X),		LCTL(KC_C),     LCTL(KC_V)
     ),
 
-    /*
-            LAYER 3 - AUDACITY
-    /-----------------------------`
-    |  TO(0)  |   REC   |   canc  |	L ENC: pan right/left
-    |---------|---------|---------|
-    |   ctrl  |   play  |  pause  |	R ENC: mouse wheel up/down
-    \-----------------------------'
-    */
+    /* LAYER 3
+     * ,-----------------------------.
+     * |  TO(0)  |    W    |    E    |	L ENC: pan right/left
+     * |---------+---------+---------|
+     * |    A    |    S    |    D    |	R ENC: mouse wheel up/down
+     * `-----------------------------'
+     */
     [3] = LAYOUT(
-                TO(0),     	LSFT(KC_R),    	KC_DEL,
-    		    KC_LCTL,	KC_SPC,         KC_P
+                TO(0),   KC_W,    KC_E,
+    		    KC_A,	 KC_S,    KC_D
     ),
 
 };
 
 
 bool encoder_update_user(uint8_t index, bool clockwise) {
-    if (index == 0) {											/* LEFT ENCODER */
+    /* LEFT ENCODER */
+    if (index == 0) {
         switch (get_highest_layer(layer_state)) {
```

Comment:
You should be using the Encoder Map feature.

https://docs.qmk.fm/#/feature_encoders?id=encoder-map

Reply:
done